package.json

```diff
@@ -29,6 +29,7 @@
     "@angular/material": "~6.3.3",
     "@angular/platform-browser": "~6.0.9",
     "@angular/router": "~6.0.9",
+    "@types/qs": "^6.9.3",
```

Comment:
Do we actually need this new npm package?

Reply:
qs is "query string" which I added during development.  It looks like it's no longer required.

Reply:
Nakul has removed it.